frontend/install.rst

```diff
@@ -79,8 +77,8 @@ directory:
 .. code-block:: bash
 
     $ cd irma-frontend/web
-    $ npm install
-    $ node_modules/.bin/bower install
+    $ sudo npm install
```

Comment:
agreed but in manual install there is only one user "irma" and without sudo it dont work as irma got no home dir